spec/mailers/candidate_mailer_offers_and_rejections_spec.rb

```diff
@@ -358,7 +358,6 @@
       'provider name' => 'Falconholt Technical College has confirmed your deferred offer to study',
       'name and code for course' => 'Forensic Science (E0FO)',
       'start date of new course' => 'June 2020',
-      'date offer was deferred' => 'This was deferred from (October 2019)',
```

Comment:
Instead of removing the test, you could test for the content you've changed. We've already tested that the correct start date is rendering on line 360, so you could just test the other little bit of text. 

You could replace the line you removed with:

` 'course starts text' => 'The course starts',`

Reply:
Thanks @avinhurry! I've added a new commit - have I done that right?

Reply:
Looks great! Good job 🚀